platform/string_storage_base.hpp

```diff
@@ -14,6 +14,7 @@ class StringStorageBase
   void Clear();
   bool GetValue(std::string const & key, std::string & outValue) const;
   void SetValue(std::string const & key, std::string && value);
+  void Update(std::map<std::string, std::string> const & values);
```

Comment:
How do you propose to _delete_ the setting remotely from the server, if it is not needed anymore?

Reply:
Just set an empty value.

Reply:
No, it should be removed completely, otherwise, users will have polluted settings.